File: app/src/main/java/com/senerunosoft/puantablosu/model/Score.java
package com.senerunosoft.puantablosu.model;

import java.util.HashMap;

public class Score {
    private int scoreOrder;
    private HashMap<String, Integer> scoreMap;

    public Score() {
    }

    public Score(int scoreOrder, HashMap<String, Integer> scoreMap) {
        this.scoreOrder = scoreOrder;
        this.scoreMap = scoreMap;
    }


    public int getScoreOrder() {
        return scoreOrder;
    }

    public HashMap<String, Integer> getScoreMap() {
        return scoreMap;
    }

    public void setScoreMap(HashMap<String, Integer> scoreMap) {
        this.scoreMap = scoreMap;
    }
}

File: app/src/main/java/com/senerunosoft/puantablosu/model/Score.kt
package com.senerunosoft.puantablosu.model

import com.google.gson.annotations.SerializedName

data class Score(
    @SerializedName("scoreOrder")
    val scoreOrder: Int = 0,
    @SerializedName("scoreMap")
    var scoreMap: HashMap<String, Int> = HashMap()
) {
    // Empty constructor for Gson
    constructor() : this(0, HashMap())
}